__tests__/users.test.js

```diff
@@ -0,0 +1,69 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const app = require('../lib/app');
+const request = require('supertest');
+const UserService = require('../lib/services/UserService');
+
+const sampleUser = {
+  first_name: 'Test',
+  last_name: 'Testerson',
+  email: 'test2@test.com',
+  password: '12345',
+};
+
+describe('users routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('POST /api/v1/users creates a new user', async () => {
+    const resp = await request(app).post('/api/v1/users').send(sampleUser);
+    expect(resp.status).toBe(200);
+    const { first_name, last_name, email } = sampleUser;
+
+    expect(resp.body).toEqual({
+      id: expect.any(String),
+      first_name,
+      last_name,
+      email,
+    });
+  });
+  it('POST /api/v1/sessions signs in an existing user', async () => {
```

Comment:
you could also post to sessions with a user that doesn't exist to verify that it won't give them a cookie if they're not in the database